src/vl-tooltip.js

```diff
@@ -352,84 +376,76 @@
   }
 
   /**
-  * Get one field value from a datum.
-  * @param {Object} datum - the datum of an item.
+  * Get a field value from a data map.
+  * @param {d3.map} itemData - a map of item.datum
   * @param {string} field - the name of the field. It can contain "." to specify
-  * that the field is not a direct child of datum.
-  * @return the field value, or undefined if the value cannot be found.
+  * that the field is not a direct child of item.datum
+  * @return the field value on success, undefined otherwise
   */
   // TODO(zening): Mute "Cannot find field" warnings for composite vis (issue #39)
-  function getValue(datum, field) {
-    if (field.includes(".")) {
-      var accessors = field.split(".");
-      var value = datum;
-      var path = "";
+  function getValue(itemData, field) {
+    var value;
+    
+    var accessors = field.split('.');
```

Comment:
No this `field. x.a` thing doesn't work in Vega-Lite (yet). I think it's handy though. I will create an issue (should I?)

```json
{
  "description": "A simple bar chart with embedded data.",
  "data": {
    "values": [
      {"obj": {"a": "A","b": 28}},
      {"obj": {"a": "A","b": 28}},
      {"obj": {"a": "B","b": 55}},
      {"obj": {"a": "C","b": 43}},
      {"obj": {"a": "D","b": 91}},
      {"obj": {"a": "E","b": 81}},
      {"obj": {"a": "F","b": 53}},
      {"obj": {"a": "G","b": 19}},
      {"obj": {"a": "H","b": 87}},
      {"obj": {"a": "I","b": 52}}
    ]
  },
  "mark": "bar",
  "encoding": {
    "x": {"field": "obj.a","type": "ordinal"},
    "y": {"field": "obj.b","type": "quantitative"}
  }
}
```